frontend/src/api/app.js

```diff
@@ -36,7 +37,9 @@ export function elementPathRequest(pathType, elementId) {
 export function queryPathsRequest(query, limit, child) {
   return axios.get(
     config.API_URL +
-      '/menu/queryPaths?nameQuery=' +
+      '/' +
+      MENU_QUERY_PATHS +
+      '?nameQuery=' +
```

Comment:
@petrican 

I've rechecked your PR and I have the following idea to solve it elegantly,
because as I was suggesting initially, after seeing it, does not look elegant at all.

* append here `&silentError=true`
* revert/remove that MENU_QUERY_PATHS constant, in that error handling function from App.js, don't check for `MENU_QUERY_PATHS`

Basically, revert your last commit and rely entirely on `&silentError=true`.
If that one is set in the query then the frontend will silent the error.
In the future that might be a good indicator for the backend too.

wdyt?

Reply:
yes. I agree that is going to be more elegant